### DISCUSSION TITLE
How to delete and resubmit the icon?

### DISCUSSION THREAD

**Discussion by shmgqn** [elrumo/macOS_Big_Sur_icons_replacements#1142] [Help with the website] [answered]
I redesigned the icon and want to delete the previously submitted version and upload the new version. How do I do this?

**Comment by rodgerwerner** [comment, marked as answer]
@elrumo has stated that he wants to add this feature in the future, but with the recent hack I would imagine this has slowed things down a tiny bit. I would think at this point if you wanted it updated sooner the best bet would be to contact @elrumo directly so it could be done manually in the mean time
